modules/nf-core/getorganelle/config/main.nf

```diff
@@ -35,7 +35,7 @@ process GETORGANELLE_CONFIG {
     def args = task.ext.args ?: ''
     """
     mkdir -p getorganelle/{LabelDatabase,SeedDatabase}
-    touch getorganelle/{LabelDatabase,SeedDatabase}/${organnel_type}.fasta
```

Comment:
Aah, this is why we test the stubs...

Reply:
Yes, I have seen quite a few broken stubs now which were never tested.